packages/riverpod_generator/README.md

```diff
@@ -156,6 +156,19 @@ class Home extends ConsumerWidget {
 }
 ```
 
+## Global configuration
+
+You can change provider name suffix in the build.yaml file:
+
+```yaml
+targets:
+  $default:
+    builders:
+      riverpod_generator:
+        options:
+          provider_name_suffix: "Provider" # (default)
```

Comment:
```suggestion
          # Could be changed to "Pod", such that riverpod_generator
          # would generate "countPod" instead of "countProvider"
          provider_name_suffix: "Provider" # (default)
```